pkg/scheduler/apis/config/validation/validation.go

```diff
@@ -216,3 +217,115 @@ func validateExtendedResourceName(name v1.ResourceName) []error {
 	}
 	return validationErrors
 }
+
+// ValidateInterPodAffinityArgs validates that InterPodAffinityArgs are correct.
+func ValidateInterPodAffinityArgs(args config.InterPodAffinityArgs) error {
+	return ValidateHardPodAffinityWeight(field.NewPath("hardPodAffinityWeight"), args.HardPodAffinityWeight)
+}
+
+const (
+	// MinHardPodAffinityWeight is the minimum HardPodAffinityWeight.
+	MinHardPodAffinityWeight int32 = 0
```

Comment:
Don't need to be exported anymore(?).

Reply:
and perhaps define them inside the validation function itself?